compiler/src/iree/compiler/Codegen/Common/GPU/GPULowerToUKernels.cpp

```diff
@@ -5,12 +5,12 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 #include "iree/compiler/Codegen/Common/GPU/Passes.h"
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h"
```

Comment:
Is this still needed here?

Reply:
Yes, for the `getLoweringConfig` helper, which is templated and so helps regardless of the fact that we use `IREE::GPU::LoweringConfigAttr`.